GOVERNANCE.md

```diff
@@ -0,0 +1,114 @@
+If the vote passes without any significant disagreements then the member can be added to the relevant team or role.
+There is no strict set of criteria for addition but generally people who are active in the community and project for an extended period of time, have contributed and added value, and share in the organization's goals & vision can be considered.
+
+### Removing members/roles
+
+This is not an action that the organization wants to perform in an ideal circumstances but this is something we do require some reference for so that people do not feel unfairly treated.
+There are a number of circumstances where this might need to be performed and this section outlines (non-exhaustively) those reasons and justifications.
+
+- **Removal requests**: If a member simply asks to be removed from a team or role for any reason then this should be done as they wish with no votes required.
+
+- **Inactivity**: If a member of a team or role becomes inactive for an extended period of time (3+ months) without any justification or conversation then removal may be required. The member should be first contacted by a another team member and, depending on the outcome, it should then be brought up and voted on as with any other decision. If the vote passes then the member will be removed from these various roles and teams. This vote may be to remove a person entirely or it may be to move to a role with fewer privileges but remain within the organization.
+
+-  **Misuse of privileges, negligence of duties or policy/code of conduct breaking**: If a member of a team or role is found to be misusing privileges granted to them or is found to have broken our policies, guidelines, rules or code of conduct then this may be grounds for removal of privileges or team/role membership. In serious cases it might be that we need to remove these privileges instantly in the case where it is believed that harm may come to the organization or project but these can be re-instated after discussion and voting if that is the outcome.
+
+## Modifying the organizational structure
+
+Sometimes it may be necessary to modify the organizational structure as defined in this document.
+This would be done in the same way as any other change as per our [Policies](https://github.com/pulsar-edit/.github/blob/main/POLICY.md) with proposals and a vote.
+If the vote passes then any new team or role additions can be performed in the same way as this document describes.
+
+## Other roles and responsibilities
+
+Not every single role or responsibility has been detailed in this document. Some responsibilities are handled as and when required and to the person or people most suited for it.
+Such areas include access details and administration of various tools and services used by the organization, social media account access, fund/donation access and expenses approval and organization secrets access.
+The above policies regarding adding or removing people to these specific privileges still apply.
```

Comment:
In all honesty, I think we don't in practice _have_ policies we go by on this stuff. We have not been voting, for instance, on who has access to the non-GitHub services we use. Cirrus is implicit via GitHub perms. etc.

This section feels "docs before the policy", "cart before horse", and I don't like being critical, but it doesn't sit right telling people one thing when I full well expect us to do another.

Reply:
> We have not been voting, for instance, on who has access to the non-GitHub services we use.

No but I think in my opinion it *should* be done - at the very least these should be (where possible) via the admin email account. For those that are scoped around "teams" type accounts then we should probably start to be more granular with those permissions.

It might seem overkill to vote on all of them in which case I'm all ears for alternative wording here.